.circleci/config.yml

```diff
@@ -101,7 +101,7 @@ jobs:
           command: bundle --path vendor/bundle
       - run:
           name: Run Unit Tests
-          command: bundle exec fastlane test_without_building name:WordPressUnitTests try_count:3 device:'<< parameters.device >>' ios-version:'<< parameters.ios-version >>'
```

Comment:
Now that the retry is configured in the Test Plan file and handled by `xcodebuild`, we don't need to manually specify one to pass down to the custom action `multi_scan`.